CONTRIBUTING.md

```diff
@@ -67,7 +67,21 @@ The default policy for taking contributions is “Squash and Merge” - because
 
 ### Make sure your patch works for all supported versions of swift
 
-The CI will do this for you.  You can use the docker-compose files included if you wish to check locally.  Currently all versions of swift >= 5.7 are supported.  For example usage of docker compose see the main [README](./README.md#an-alternative-using-docker-compose)
+The CI will do this for you.  Currently all versions of swift >= 5.8 are supported.
+
+If you wish to test this locally you have two options [act](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://github.com/nektos/act) and docker compose files.
+
+#### Act
+
+[Install act](https://nektosact.com) and then you can run the full suite of checks via:
+```
+act pull_request
+```
+Note that NIOSwift matrix testing makes use of nightly builds, so you may want to make use of the ```--action-offline-mode``` to avoid repulling those.
```

Comment:
```suggestion
Note that SwiftNIO matrix testing makes use of nightly builds, so you may want to make use of the ```--action-offline-mode``` to avoid repulling those.
```